roles/edpm_ovn_bgp_agent/meta/argument_specs.yml

```diff
@@ -3,4 +3,127 @@ argument_specs:
+        type: int
+        default: 300
+      edpm_ovn_bgp_agent_expose_tenant_networks:
+        description: Exposing tenant networks.
+        type: bool
+        default: false
+      edpm_ovn_bgp_agent_expose_ipv6_gua_tenant_networks:
+        description: Exposing tenant networks with IPv6 addresses.
+        type: bool
+        default: false
+      edpm_ovn_bgp_agent_driver:
+        type: str
+        default: ovn_bgp_driver
+      edpm_ovn_bgp_agent_private_key:
+        description: Location of private TLS key. Keep it safe.
+        type: path
+        default: /etc/pki/tls/private/ovn_bgp_agent.key
+      edpm_ovn_bgp_agent_certificate:
+        description: Location of TLS certificate used by BGP agent.
+        type: path
+        default: /etc/pki/tls/certs/ovn_bgp_agent.crt
+      edpm_ovn_bgp_agent_ca_cert:
+        description: Location of IPA certificate.
+        type: path
+        default: /etc/ipa/ca.crt
+      edpm_ovn_bgp_agent_internal_tls_enable:
+        description: Enable internal TLS.
+        type: bool
+        default: false
+      edpm_ovn_bgp_agent_config_basedir:
+        description: Location of BGP agent configuration base directory.
+        type: path
+        default: "/var/lib/config-data/ansible-generated/ovn-bgp-agent"
+      edpm_ovn_bgp_agent_bgp_as:
+        description: BGP Autonomous System number
+        type: int
+        default: 64999
+      edpm_ovn_bgp_agent_clear_vrf_routes_on_startup:
+        type: bool
+        default: false
+      edpm_ovn_bgp_agent_bgp_nic:
+        type: str
+        default: bgp-nic
+      edpm_ovn_bgp_agent_bgp_vrf:
+        type: str
+        default: bgp-vrf
+      edpm_ovn_bgp_agent_bgp_vrf_table_id:
+        description: ID of Virtual Routing and Forwarding table.
+        type: int
+        default: 10
+      edpm_ovn_bgp_agent_root_helper:
+        description: Root helper for BGP agent.
+        type: str
+        default: "sudo ovn-bgp-agent-rootwrap /etc/ovn-bgp-agent/rootwrap.conf"
+      edpm_ovn_bgp_agent_root_helper_daemon:
+        description: Root helper for BGP agent as a daemon.
+        type: str
+        default: "sudo ovn-bgp-agent-rootwrap-daemon /etc/ovn-bgp-agent/rootwrap.conf"
+      edpm_ovn_bgp_agent_ovsdb_connection:
+        description: Connection to the OVS database, including port and protocol.
+        type: str
+        default: "tcp:127.0.0.1:6640"
+      edpm_ovn_bgp_agent_ovs_manager:
+        type: str
+        default: "ptcp:6640:127.0.0.1"
+      edpm_ovn_bgp_agent_image:
+        type: str
+        default: "quay.io/podified-antelope-centos9/openstack-ovn-bgp-agent:current-podified"
+      edpm_ovn_bgp_agent_address_scopes:
+        description: Optional parameter
+        type: str
+        default: ''
+      edpm_ovn_bgp_agent_bgp_router_id:
+        description: Optional parameter
+        type: str
+        default: ''
+      edpm_ovn_bgp_agent_evpn_local_ip:
+        description: Optional parameter
```

Comment:
`description: IP address of local EVPN VXLAN (tunnel) endpoint. This option can be used instead of the evpn_nic one. If none specified, it will take the one from the loopback device.`